app/controllers/api/v1/requests_controller.rb

```diff
@@ -12,6 +12,10 @@ def create
         render json: @request
       end
 
+      def show
+        render json: Request.unclaimed.find_by(id: params[:id])
```

Comment:
I think removing the `unclaimed` check could make this action a little friendier to other use cases going forward.